code/game/objects/items/weapons/mine.dm

```diff
@@ -112,8 +121,10 @@
 /obj/item/weapon/mine/Crossed(mob/AM)
 	if (armed)
 		if (isliving(AM))
+			if (AM.faction == "excelsior")
```

Comment:
There's defines with that string like ROLE_EXCELSIOR or similar, other added lines should be updated to reflect that including neutral faction

Reply:
Fair enough, I changed all the faction checks to antag_id checks and used excelsior_rev which is the string for the ROLE_EXCELSIOR define. Hopefully that works for you?